AceComm-3.0/AceComm-3.0.lua

```diff
@@ -116,24 +119,17 @@ function AceComm:SendCommMessage(prefix, text, distribution, target, prio, callb
 		-- fits all in one message
 		CTL:SendAddonMessage(prio, prefix, text, distribution, target, queueName, ctlCallback, textlen)
 	else
-		maxtextlen = maxtextlen - 1	-- 1 extra byte for part indicator in prefix(4.0)/start of message(4.1)
-
-		-- first part
-		local chunk = strsub(text, 1, maxtextlen)
-		CTL:SendAddonMessage(prio, prefix, MSG_MULTI_FIRST..chunk, distribution, target, queueName, ctlCallback, maxtextlen)
-
-		-- continuation
-		local pos = 1+maxtextlen
-
-		while pos+maxtextlen <= textlen do
-			chunk = strsub(text, pos, pos+maxtextlen-1)
-			CTL:SendAddonMessage(prio, prefix, MSG_MULTI_NEXT..chunk, distribution, target, queueName, ctlCallback, pos+maxtextlen-1)
-			pos = pos + maxtextlen
+		local chunkCount = tostring(ceil(#text / maxtextlen))
```

Comment:
Eh this is wrong, it needs to be recalculated after maxtextlen is adjusted below, and after chunkCount is updated maxtextlen might change.